br.com.gualandi.Xjump.yml

```diff
@@ -0,0 +1,20 @@
+app-id: br.com.gualandi.Xjump
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: xjump
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+modules:
+  - name: xjump
+    buildsystem: simple
+    build-commands:
+      - ./configure --prefix=/app
+      - make
+      - make install
```

Comment:
```suggestion
```

autotools is the default and should just work.

Reply:
The project technically doesn't use autotools, it is a custom configure script. Can I still use "buildsystem:autotools" in that case?

Reply:
should be fine and is very easy to test ;-)

Reply:
Indeed, I tested it here and the 'autotools' setting invoked passed exactly the same flags to ./configure. I have pushed a commit to switch to that. Thanks for the tip!